cogapp/test_cogapp.py

```diff
@@ -1046,7 +1053,7 @@ def test_print_output(self):
         make_files(d)
         self.cog.callable_main(["argv0", "-rP", "test.cog"])
         self.assertFilesSame("test.cog", "test.out")
-        output = self.output.getvalue()
+        output, errput = self.capsys.readouterr()
```

Comment:
throughout: if we aren't using `errput`, we should assign `output, _ = ...`, or we should make assertions about errput.